components/gfx/display_list/mod.rs

```diff
@@ -76,6 +76,51 @@ impl OpaqueNode {
     }
 }
 
+/// LayerInfo is used to store information PaintLayer metadata during DisplayList construction.
```

Comment:
Oh, thanks for catching this. I think I meant to write:
"LayerInfo is used to store PaintLayer metadata during DisplayList construction."